app/javascript/shared/helpers/AudioNotificationHelper.js

```diff
@@ -47,6 +47,10 @@ export const shouldPlayAudio = (
   userId,
   isDocHidden
 ) => {
+  if (!isDocHidden) {
```

Comment:
If the tab is not hidden the audio alert will not be available and the favicon badge will be shown for a new message.